flow-server/src/test/java/com/vaadin/flow/router/QueryParametersTest.java

```diff
@@ -235,6 +235,106 @@ public void shortHands() {
         Assert.assertTrue(qp1.getParameters("bar").isEmpty());
     }
 
+    @Test
+    public void excluding() {
+        Map<String, List<String>> paramMap = new HashMap<>();
+        paramMap.put("one", Collections.singletonList("1"));
+        paramMap.put("two", Collections.singletonList("2"));
+        paramMap.put("three", Collections.singletonList("3"));
+
+        QueryParameters params = new QueryParameters(paramMap);
+        QueryParameters newParams = params.excluding("two");
+
+        Assert.assertEquals(2, newParams.getParameters().size());
+        Assert.assertEquals(Collections.singletonList("1"),
+                newParams.getParameters("one"));
+        Assert.assertEquals(Collections.emptyList(),
+                newParams.getParameters("two"));
+        Assert.assertEquals(Collections.singletonList("3"),
+                newParams.getParameters("three"));
+    }
+
+    @Test
+    public void excludingNone() {
+        Map<String, List<String>> paramMap = new HashMap<>();
+        paramMap.put("one", Collections.singletonList("1"));
+        paramMap.put("two", Collections.singletonList("2"));
+        paramMap.put("three", Collections.singletonList("3"));
+
+        QueryParameters params = new QueryParameters(paramMap);
+        QueryParameters newParams = params.excluding();
+
+        Assert.assertEquals(params, newParams);
+    }
+
+    public void including() {
```

Comment:
`@Test` annotation is missing
```suggestion
    @Test
    public void including() {
```

Reply:
Done